src/foam/nanos/auth/AuthService.js

```diff
@@ -109,6 +109,18 @@ foam.INTERFACE({
         }
       ]
     },
+    {
+      name: 'validatePassword',
```

Comment:
How do you know what's wrong with the password if it doesn't like it? An Exception (which is reasonable)? But if that's the case, then we don't really need a return value do we? Validity as implied but just returning without an exception.

Reply:
Done .